resources/themes/cura-light/icons/low/CheckThin.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Hmm, but now nothing uses this `CheckThin` file, right?

If it's not used, I would say we remove this. Not even plug-ins are using it since they would need to use the new name then.

Reply:
CheckThin is being used by the tick version of this icon. The component is defined in Uranium.
<img width="400" alt="Screenshot 2022-01-28 at 17 16 28" src="https://user-images.githubusercontent.com/23387864/151582572-ea26d4c3-47ac-4f52-bf9f-ec873e3dbe88.png">
e check badge pop up in Uranium

Reply:
Ah, I probably had the wrong branch checked out on Uranium when I checked this then, since it's a different repository.